.rubocop.yml

```diff
@@ -7,6 +7,10 @@ Style/FrozenStringLiteralComment:
 Style/ModuleFunction:
   EnforcedStyle: extend_self
 
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    '%i': ()
```

Comment:
It generates arrays, so IMHO the default (prefer square brackets) is superior.

Also, we need an official Rubocop file :)